src/libs/actions/connections/NetSuiteCommands.ts

```diff
@@ -0,0 +1,274 @@
+import type {OnyxUpdate} from 'react-native-onyx';
+import Onyx from 'react-native-onyx';
+import type {ValueOf} from 'type-fest';
+import * as API from '@libs/API';
+import {WRITE_COMMANDS} from '@libs/API/types';
+import * as ErrorUtils from '@libs/ErrorUtils';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {Connections} from '@src/types/onyx/Policy';
+
+function updateNetSuiteOnyxData<TSettingName extends keyof Connections['netsuite']['options']['config']>(
```

Comment:
Nice 💯

Reply:
Just curious, how do we imagine this working with the deeper settings like the `syncOptions`? If we plan on using a different method for those maybe we should call this `updateNetSuiteConfigOnyxData`